src/transforms/transform-deployment.ts

```diff
@@ -155,6 +157,34 @@ export function mapEnvironment(environment: string): string {
 	return jiraEnv;
 }
 
+// Maps commit summaries to an array of a single association containing the commit keys (commit hash and repository id)
+const mapCommitSummariesToAssociationArray = (
+	repositoryId: string,
+	commitSummaries?: CommitSummary[]
+): JiraAssociation[] | undefined => {
+
+	let associationArray;
+
+	if (commitSummaries && commitSummaries.length) {
+		const commitKeys = commitSummaries
+			.map((commitSummary) => {
+				return {
+					commitHash: commitSummary.sha,
+					repositoryId: repositoryId
+				};
+			});
+
+		associationArray = [
+			{
+				associationType: "commit",
+				values: commitKeys
+			}
```

Comment:
Documentation for this says:

> associationType REQUIRED
> string
> Defines the asssociation type.
> Valid values: issueKeys, issueIdOrKeys

Or is `commit` a new type that the documentation hasn't been updated with?

Reply:
`commit` is a new type that we're adding. Documentation is due to be updated later in the project.

Reply:
Cool, in that case, can you please update the type for `associationType` to include the possible values and not just a generic string?  Thanks.